include/net/net_pkt.h

```diff
@@ -2288,8 +2288,13 @@ static inline void *net_pkt_cursor_get_pos(struct net_pkt *pkt)
  * Note: net_pkt's cursor should be properly initialized
  *       Cursor position will be updated after the operation.
  *       Depending on the value of pkt->overwrite bit, this function
- *       will affect the buffer length or not: if it's 0, skip will
- *       actually apply the move in the buffer as it had written in it.
+ *       will affect the buffer length or not. If it's true, it will
```

Comment:
I still find this behavior and/or description confusing. Most likely "overwrite" name for that flag isn't the best. Anyway, just reciting aloud, will be looking at the code later and will see if I can come up with specific suggestions.